docs/ADDING_MACHINES.md

```diff
@@ -146,6 +178,65 @@ With KubeJS, the texture can go in the following folders respectively:
 - `kubejs/assets/modern_industrialization/textures/blocks/machines/pyrolyse_overlays/overlay_front.png`.
 - `kubejs/assets/modern_industrialization/textures/blocks/machines/pyrolyse_overlays/overlay_front_active.png`.
 
+### Extra multiblock configuration options
+Both steam and electric have an optional config function parameter that can be added to the end for further customization. 
+```
+    event.simpleSteamCraftingMultiBlock(
+        /* GENERAL PARAMETERS */
+        // English name, internal name, recipe type, multiblock shape
+        "Pyrolyse Oven", "pyrolyse_oven", PYROLYSE_OVEN, pyrolyseShape,
+        /* REI DISPLAY CONFIGURATION */
+        // REI progress bar
+        event.progressBar(77, 33, "arrow"),
+        // REI item inputs, item outputs, fluid inputs, fluid outputs
+        itemInputs => itemInputs.addSlots(56, 35, 1, 2), itemOutputs => itemOutputs.addSlot(102, 35),
+        fluidInputs => fluidInputs.addSlot(36, 35), fluidOutputs => fluidOutputs.addSlot(122, 35),
+        /* MODEL CONFIGUATION */
+        // casing of the controller, overlay folder, front overlay?, top overlay?, side overlay?
+        "heatproof_machine_casing", "pyrolyse_overlays", true, false, false,
+        /* OPTIONAL CONFIGURATION */
+        config => {}
+    );
+})
+```
+Methods that config exposes:
+
+### `config.steamCustomOverclock()`
+* Only on steam multiblocks, allows changing the overclock amount and duration for the given items or blocks.
+* Use `steamCustomOverclock({})` to disable the default gunpowder overclocking behavior.
+* If multiple items are contributing to the machines overclock, the highest multiplier is processed first.
+* Items with the same multiplier value have their ticks summed together
+
+Example:
```

Comment:
Can we put this explanation above in the normal machine section, and only refer to it here. Something like

```
* Only on steam multiblocks, allows changing the overclock amount and duration for the given items or blocks.
* See above, same as single block machines.
```

This is to avoid duplication of info.